### PR TITLE
Add swift SDK doc

### PR DESCRIPTION
**Pull request by dmipon**

### CODE REVIEW

**Comment by christoffer-a-sinch** [comment]
LGTM, lets just coordinate with Jorge so that we don't publish this before we actually have the new Swift framework packages in place at downloads.sinch.com etc.

File: docs/voice/voice-ios-cloud/voice-ios-cloud-swift-sdk.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+- _Privacy - Microphone Usage Description_ (`NSMicrophoneUsageDescription`):
+
+  [NSMicrophoneUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nsmicrophoneusagedescription) describes the reason your app accesses the microphone. When the system prompts the user to allow access, this string is displayed as part of the alert, and it _cannot_ be left empty.
+
+In addition to the keys above, if video calling functionality will be enabled and used, add the following to your _Info.plist_:
+
+- _Privacy - Camera Usage Description_ (`NSCameraUsageDescription`):
+
+  [NSCameraUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nscamerausagedescription) describes the reason that your app accesses the camera. When the system prompts the user to allow access, this string is displayed as part of the alert, and it _cannot_ be left empty.
```
Comment (GiovanniBeriSinch) [inline docs/voice/voice-ios-cloud/voice-ios-cloud-swift-sdk.md:44]:
```suggestion
  [NSCameraUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nscamerausagedescription) describes the reason your app accesses the camera. When the system prompts the user to allow access, this string is displayed as part of the alert, and it _cannot_ be left empty.
```

**Review by jsiqueir-sinch — COMMENTED** [review]
I would change: "You need to specify which one you would like to obtain." 
for
"Once you signup as a Beta customer, you will have access to both versions of our IOS SDKs".

Don't see any advantage by restricting the SDK platform customer access.  Let them play with both..

**Comment by dmipon** [comment]
@jsiqueir-sinch fixed

**Pull request merged**